internal/rbd/nodeserver.go

```diff
@@ -1042,8 +1067,18 @@ func (ns *NodeServer) NodeExpandVolume(
 	volumePath := req.GetStagingTargetPath()
 	if volumePath == "" {
 		// If Kubernetes version < v1.19.0 the volume_path would be
-		// having the staging_target_path information
-		volumePath = req.GetVolumePath()
+		// having the staging_target_path information.
+		// stagingTargetPath is optional parameter in NodeExpandVolumeRequest request
+		// Kubernetes will not send the staging_target_path in the volume_path in some cases
+		// Refer https://github.com/kubernetes/kubernetes/issues/115343
+		var err error
+		volumePath, err = ns.getStagingPath(volumePath)
+		if err != nil {
+			// If stagingTargetPath is not found in volumePath then use
+			// volumePath
+			volumePath = req.GetVolumePath()
+			log.ErrorLog(ctx, "failed to find stagingTargetPath from volumePath %v: %v", volumePath, err)
```

Comment:
This isn't always an error, and there is an alternative value used... Maybe log as info?